packages/peregrine/package.json

```diff
@@ -32,6 +32,7 @@
     "@apollo/react-hooks": "~3.1.2",
     "@babel/runtime": "~7.4.2",
     "apollo-client": "~2.6.4",
+    "graphql-tag": "~2.10.1",
```

Comment:
Now we have some query code written directly in talons so this is a peer dep.

Reply:
The queries in question are runtime (so a client can change filters without having to redeploy) introspection queries. I can try moving them to the components though.